contracts/main/bridge/FathomBridge.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "./AsterizmClientUpgradeableTransparency.sol";
+import "../interfaces/IStablecoinAdapter.sol";
+import "../interfaces/IBookKeeper.sol";
+import "../interfaces/ICagable.sol";
+import "../interfaces/IFathomBridge.sol";
+import "../utils/SafeToken.sol";
+
+/**
+ * @title FathomBridge
```

Comment:
Let's move NatSpec docs to the interface. It will help with readability

Reply:
Okay, makes sense.